packages/enzyme-matchers/src/assertions/toHaveState.js

```diff
@@ -54,18 +52,21 @@ function toHaveState(
   // error if some state doesn't exist
   if (results.missingKeys.length) {
     const missingKeys = results.missingKeys.join(', ');
+    const _key_ = results.missingKeys.length === 1 ? 'prop' : 'props';
     return {
       pass: false,
-      message: `Expected <${wrapperName}> component state to have keys of "${missingKeys}"`,
-      negatedMessage: `Expected <${wrapperName}> component state to not contain a key of "${missingKeys}".`,
+      message: `Expected the state for <${wrapperName}> to contain values for ${_key_} "${missingKeys}", but it did not.`,
+      negatedMessage: `Expected the state for <${wrapperName}> to not contain values for ${_key_} "${missingKeys}", but it did.`,
```

Comment:
i'm confused why the word `prop | props` is part of this state assertion? It would read something like:

`Expected the state for <User> to contain values for prop "id", but it did not.` 

Instead of `prop | props` should it be `key | keys` ?

Reply:
Agh! I made the same mistake I set out to fix. Updated.